### PR TITLE
Remove code that disables wifi

### PR DESCRIPTION
**Pull request by latin-panda**
**Description**

This PR:
- Remove code that disables WiFi and that retries requests to Webapp, if there is an exception from last request.
   - Because it's not possible anymore to disable the WiFi in Android 10+
   - The retry request line of code in this section was removed, to keep it simple because Wakefull service will automatically poll from Webapp again after 1 min and resend any pending item.

**Ticket:** https://github.com/medic/medic-gateway/issues/135

**License**
The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: src/main/java/medic/gateway/alert/WebappPoller.java [hunk 1]
```diff
@@ -61,41 +65,44 @@ public SimpleResponse pollWebapp() throws JSONException, MalformedURLException {
 		return response;
 	}
 
-	public Boolean moreMessagesToSend() {
```
Comment (latin-panda) [inline src/main/java/medic/gateway/alert/WebappPoller.java:64]: This function has the new change, the rest in this file is about fixing spaces and parentheses. 

It'll stop polling from webapp if there's an error. Wakefull will poll again automatically in 1 min and send any pending item. Previously discussed [here](https://github.com/medic/medic-gateway/pull/161/files#r468959249)

File: src/main/java/medic/gateway/alert/WakefulService.java [hunk 2]
```diff
@@ -32,40 +26,21 @@ public WakefulService() {
...
-					if(exResponse.ex instanceof SocketTimeoutException ||
-							exResponse.ex instanceof UnknownHostException ||
-							exResponse.ex instanceof ConnectException ||
-							exResponse.ex instanceof NoRouteToHostException) {
-						wifiMan = new WifiConnectionManager(ctx);
-						if(wifiMan.isWifiActive()) {
-							logEvent(ctx, "Disabling wifi and then retrying poll...");
-							enableWifiAfterWork = true;
-							wifiMan.disableWifi();
-							lastResponse = poller.pollWebapp();
```
Comment (latin-panda) [inline src/main/java/medic/gateway/alert/WakefulService.java:63]: Not retring again if there's an exception since Wakeful service will poll again in 1 min

**Comment by latin-panda** [comment]
Hi @garethbowen, when you have a chance, can you please review this one? I removed the WiFi logic as discuss before, is this okay?  
Thanks!

**Review by garethbowen — APPROVED** [review]
LGTM! Nice work.

**Comment by newtewt** [comment]
Can we remove the permission to change_wifi_state? 

https://github.com/medic/medic-gateway/blob/4251ff3a3a2330ad3c7a7533218174c97b97975d/src/main/AndroidManifest.xml#L20

**Comment by latin-panda** [comment]
@newtewt Good catch Nick! Thank you! 
I've removed the 2 permissions associated with this feature: CHANGE_WIFI_STATE and ACCESS_NETWORK_STATE 
and did a smoke test, looking good. 
Would you like test again before I merge?

**Comment by newtewt** [comment]
I'll run through it quickly this morning.

**Comment by newtewt** [comment]
@latin-panda  I think it's good to merge.

**Pull request merged**